workspaces/ai-integrations/plugins/ai-experience/src/components/NewsPage/NewsPage.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { makeStyles } from '@material-ui/core';
+import { Theme } from '@mui/material/styles';
+import { Page, Content } from '@backstage/core-components';
+import { NewsHeader } from './NewsHeader';
+import { NewsGrid } from './NewsGrid';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  content: {
+    backgroundColor: theme.palette.background.default,
+  },
+}));
+
+export const NewsPage = () => {
+  const classes = useStyles();
+  return (
+    <Page themeId="ai-experience">
+      <NewsHeader />
+      <Content noPadding className={classes.content}>
```

Comment:
I dont think adding background styles are necessary.

```suggestion
      <Content noPadding>
```